### ISSUE TITLE
Wallaby.js doesn't update core while running (with AVA tests)

### ISSUE THREAD

**Issue by danzel** [wallabyjs/public#753]
### Issue description or question

Since switching to use AVA tests, I've found that when the "Updating Wallaby JS core" message appears, it just sits around and doesn't finish.
Only if I stop running wallabyjs (ctrl-shift-=, stop) will the update complete.

### Wallaby.js configuration file

```javascript
module.exports = function (w) {

	return {
		files: [
			'src/**/*.ts',
			'test/utils/**/*.ts'
		],

		tests: [
			'test/**/*Spec.ts'
		],

		testFramework: 'ava',
		env: {
			type: 'node'
		}
	};
};
```

### Code editor or IDE name and version

Visual Studio Code v1.4.0

### OS name and version

Windows

**Comment by ArtemGovorov** [comment]
Thanks for the report.

Wallaby.js for VS Code has always been doing the actual update when wallaby.js is stopped. It's just perhaps before using AVA you didn't notice it because wallaby was getting the update and installing it while you had it stopped. 

There was an issue with the `Updating Wallaby JS core` message though. It was displayed when wallaby would discover that there's a newer version, as opposed to displaying it right before the update. So it was incorrectly showing the message and waiting until the stop to apply the update. Now it's fixed the extension v1.0.21 and the message gets displayed right before applying the update, so it's not confusing anymore.

**Comment by danzel** [comment]
Cool, thanks for the news :)
